Application Security Landing Page (#12937)
DataDog/documentation: content/en/security_platform/application_security/_index.md

---
title: Application Security
kind: documentation

> is_beta: true

Author: ```suggestion
description: Monitor threats targeting production system, leveraging the execution context provided by distributed traces.
is_beta: true
```

disable_sidebar: true
further_reading:
- link: "/security_platform/guide/how-appsec-works/"
  tag: "Guide"
  text: "Learn how Application Security works"
- link: "/security_platform/application_security/setup_and_configure/#compatibility"
  tag: "Documentation"
  text: "Learn more about language and framework compatibility"
---

<div class="alert alert-warning">
Application Security is currently in public beta. See the <a href="https://app.datadoghq.com/security/appsec">in-app instructions</a> to get started.
</div>

{{< img src="/security_platform/application_security/app-sec-landing-page.png" alt="The Application Security landing page in Datadog, which displays your services, suspicious requests, and detected signals" width="75%">}}

> Datadog Application Security provides protection against application-level threats by identifying and blocking attacks that target code-level vulnerabilities, such as SQL injections and cross-site scripting (XSS) exploits.

Author: ```suggestion
Datadog Application Security monitors application-level threats to identify attacks that target code-level vulnerabilities, such as Server-Side-Request-Forgery (SSRF), SQL injection, Log4Shell, and Reflected Cross-Site-Scripting (XSS).
```

Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever a vulnerability or attack occurs.

> For example, if a threat is detected, a security signal is generated in Datadog, which triggers notifications in-app and for your configured apps or services. Your team can access these signals in Datadog and begin investigation by leveraging the deep observability data provided by Application Security and APM distributed tracing in one panel.

Reviewer: > a security signal is generated in Datadog
Only for HIGH/CRITICAL, otherwise it's noise

> which triggers notifications in-app and for your configured apps or services
I don't get this sentence. Notifs can be sent out to Slack or email, or PagerDuty for instance

> For example, if a threat is detected, a security signal is generated in Datadog, which triggers notifications in-app and for your configured apps or services. Your team can access these signals in Datadog and begin investigation by leveraging the deep observability data provided by Application Security and APM distributed tracing in one panel.

Author: ```suggestion
For example, if a `HIGH` or `CRITICAL` threat is detected, a security signal is generated in Datadog. Notifications for these signals can be sent to Slack or email, or Pagerduty, for instance. Your team can access these signals in Datadog and begin investigation by leveraging the deep observability data provided by Application Security and APM distributed tracing in one panel.
```

## Configure your environment

> Powered by provided [out-of-the-box rules][3], Application Security detects threats without the manual configuration of rules for an Agent or provider. If you're already using Datadog [APM][1], setup only requires implementation of one environment variable to get started.

Reviewer: * detection rules
*of rules for an Agent or provider: not clear nor useful to me.

> If you're already using Datadog [APM][1], setup only requires implementation of one environment variable to get started.
Env variable are set, not technically implemented

> Powered by provided [out-of-the-box rules][3], Application Security detects threats without the manual configuration of rules for an Agent or provider. If you're already using Datadog [APM][1], setup only requires implementation of one environment variable to get started.

Author: ```suggestion
Powered by provided [out-of-the-box rules][3], Application Security detects threats without manual configuration. If you're already using Datadog [APM][1], setup only requires setting one environment variable to get started.
```

> To start configuring your environment to detect threats with Application Security, follow the [in-app instructions][4] and supplemental [Getting Started documentation][5]. Once configured, you can also create and configure your own [custom signal rules][6] to tailor detection methods to your environment. Once a threat is detected, begin investigating and remediating security signals in Datadog.

Reviewer: > To start configuring your environment to detect threats with Application Security, follow the [in-app instructions][4] and supplemental [Getting Started documentation][5]

I think they have to follow one or the other, as they'll likely mirror the same content.

> Once configured, you can also create and configure your own [custom signal rules][6] to tailor detection methods to your environment. 
To detect threats custom to your business-logic

> To start configuring your environment to detect threats with Application Security, follow the [in-app instructions][4] and supplemental [Getting Started documentation][5]. Once configured, you can also create and configure your own [custom signal rules][6] to tailor detection methods to your environment. Once a threat is detected, begin investigating and remediating security signals in Datadog.

Author: ```suggestion
To start configuring your environment to detect threats with Application Security, follow the [Getting Started documentation][5]. Once configured, you can begin investigating and remediating security signals in the [Security Signals Explorer][7].
```

## Investigate and remediate security signals

> Security signals are available for triage in the [Security Signals Explorer][7]. Click on any security signal to see what happened and the suggested steps to remediate the issue. In the same panel, view traces with their correlated attack flow and request information to gain further context.

Author: ```suggestion
In the [Security Signals Explorer][7], click on any security signal to see what happened and the suggested steps to remediate the issue. In the same panel, view traces with their correlated attack flow and request information to gain further context.
```

---
title: Application Security
kind: documentation
description: Monitor threats targeting production system, leveraging the execution context provided by distributed traces.
is_beta: true
disable_sidebar: true
further_reading:
- link: "/security_platform/guide/how-appsec-works/"
  tag: "Guide"
  text: "Learn how Application Security works"
- link: "/security_platform/application_security/setup_and_configure/#compatibility"
  tag: "Documentation"
  text: "Learn more about language and framework compatibility"
---

<div class="alert alert-warning">
Application Security is currently in public beta. See the <a href="https://app.datadoghq.com/security/appsec">in-app instructions</a> to get started.
</div>

{{< img src="/security_platform/application_security/app-sec-landing-page.png" alt="The Application Security landing page in Datadog, which displays your services, suspicious requests, and detected signals" width="75%">}}

> Datadog Application Security monitors application-level threats to identify attacks that target code-level vulnerabilities, such as Server-Side-Request-Forgery (SSRF), SQL injection, Log4Shell, and Reflected Cross-Site-Scripting (XSS).

Reviewer: attacks aiming to exploit code-level vulnerabilities

> Datadog Application Security monitors application-level threats to identify attacks that target code-level vulnerabilities, such as Server-Side-Request-Forgery (SSRF), SQL injection, Log4Shell, and Reflected Cross-Site-Scripting (XSS).

Author: ```suggestion
Datadog Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities, such as Server-Side-Request-Forgery (SSRF), SQL injection, Log4Shell, and Reflected Cross-Site-Scripting (XSS).
```

> Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever a vulnerability or attack occurs.

Reviewer: An attack targets your production system, or a vulnerability is triggered from the code

> Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever a vulnerability or attack occurs.

Author: ```suggestion
Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever an attack targets your production system, or a vulnerability is triggered from the code.
```

---

<div class="alert alert-warning">
Application Security is currently in public beta. See the <a href="https://app.datadoghq.com/security/appsec">in-app instructions</a> to get started.
</div>

{{< img src="/security_platform/application_security/app-sec-landing-page.png" alt="The Application Security landing page in Datadog, which displays your services, suspicious requests, and detected signals" width="75%">}}

Datadog Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities, such as Server-Side-Request-Forgery (SSRF), SQL injection, Log4Shell, and Reflected Cross-Site-Scripting (XSS).

Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever an attack targets your production system, or a vulnerability is triggered from the code.

If a `HIGH` or `CRITICAL` threat is detected with Application Security, a security signal is generated in Datadog. Notifications for these signals can be sent to Slack, email, or Pagerduty to notify your team and gain context of threats in real-time.

> View a security signal in Datadog to leverage the deep observability data provided by Application Security and APM distributed tracing. Speed up the investigation process by analyzing attack flows, viewing flame graphs, and reviewing correlated trace and log data all within one panel. Determine the cause of a threat and view remediation in the same view to eliminate context switching.

Author: ```suggestion
View a security signal in Datadog to leverage the deep observability data provided by Application Security and APM distributed tracing. Speed up the investigation process by analyzing attack flows, viewing flame graphs, and reviewing correlated trace and log data all within one panel. Determine the cause of a threat and view remediation steps in the same view to eliminate context switching.

With Application Security, you can create queries to cut through the noise of continuous trace data to focus solely on securing and protecting your environment as threats arise.
```

---
title: Application Security
kind: documentation
description: Monitor threats targeting production system, leveraging the execution context provided by distributed traces.
is_beta: true
disable_sidebar: true
further_reading:
- link: "/security_platform/guide/how-appsec-works/"
  tag: "Guide"

>   text: "Learn how Application Security works"

Reviewer 2: I don't think this will be there for the release. Maybe instead, do we know what the Release Notes link will be? I think we're supposed to add that to landing pages now.

Author: Good question, @arnaudbreton do we have a link handy for the release notes? If not, we can add after publishing to direct folks to more details.

- link: "/security_platform/application_security/setup_and_configure/#compatibility"
  tag: "Documentation"
  text: "Learn more about language and framework compatibility"
---

<div class="alert alert-warning">

> Application Security is currently in public beta. See the <a href="https://app.datadoghq.com/security/appsec">in-app instructions</a> to get started.

Reviewer 2: ```suggestion
Application Security is in public beta. See the <a href="https://app.datadoghq.com/security/appsec">in-app instructions</a> to get started.
```

</div>

> {{< img src="/security_platform/application_security/app-sec-landing-page.png" alt="The Application Security landing page in Datadog, which displays your services, suspicious requests, and detected signals" width="75%">}}

Author: ```suggestion
{{< img src="/security_platform/application_security/app-sec-landing-page.png" alt="A security signal panel in Datadog, which displays attack flows and flame graphs" width="75%">}}
```

Datadog Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities, such as Server-Side-Request-Forgery (SSRF), SQL injection, Log4Shell, and Reflected Cross-Site-Scripting (XSS).

Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever an attack targets your production system, or a vulnerability is triggered from the code.

> For example, when a `HIGH` or `CRITICAL` threat is detected, a security signal is generated in Datadog. Notifications for these signals can be sent to Slack, email, or Pagerduty to notify your team and gain context of threats in real-time.

Reviewer 2: ```suggestion
For example, when a `HIGH` or `CRITICAL` threat is detected, a security signal is generated in Datadog. Notifications for these signals can be sent to Slack, email, or PagerDuty to notify your team and gain context of threats in real-time.
```

> If a `HIGH` or `CRITICAL` threat is detected with Application Security, a security signal is generated in Datadog. Notifications for these signals can be sent to Slack, email, or Pagerduty to notify your team and gain context of threats in real-time.

Author: ```suggestion
For example, when a `HIGH` or `CRITICAL` threat is detected, a security signal is generated in Datadog. Notifications for these signals can be sent to Slack, email, or Pagerduty to notify your team and gain context of threats in real-time.
```

> View a security signal in Datadog to leverage the deep observability data provided by Application Security and APM distributed tracing. Speed up the investigation process by analyzing attack flows, viewing flame graphs, and reviewing correlated trace and log data all within one panel. Determine the cause of a threat and view remediation steps in the same view to eliminate context switching.

Author: ```suggestion
Instantly transition to investigating a security signal in Datadog to begin investigation. Leverage the deep observability data provided by Application Security and APM distributed tracing to resolve application issues in one view. Analyze attack flows, view flame graphs, and review correlated trace and log data to pinpoint application vulnerabilities. Eliminate context switching by flowing through application data into remediation steps, all within the same panel.
```

> From onset of a threat, quickly pivot to investigate a security signal in Datadog. Leverage the deep observability data provided by Application Security and APM distributed tracing, in one view, to resolve application issues. Analyze attack flows, view flame graphs, and review correlated trace and log data to pinpoint application vulnerabilities. Eliminate context switching by flowing through application data into remediation steps, all within the same panel.

Reviewer 2: ```suggestion
From the onset of a threat, quickly pivot to investigate a security signal in Datadog. Leverage the deep observability data provided by Application Security and APM distributed tracing, in one view, to resolve application issues. Analyze attack flows, view flame graphs, and review correlated trace and log data to pinpoint application vulnerabilities. Eliminate context switching by flowing through application data into remediation steps, all within the same panel.
```

> Instantly transition to investigating a security signal in Datadog to begin investigation. Leverage the deep observability data provided by Application Security and APM distributed tracing to resolve application issues in one view. Analyze attack flows, view flame graphs, and review correlated trace and log data to pinpoint application vulnerabilities. Eliminate context switching by flowing through application data into remediation steps, all within the same panel.

Author: ```suggestion
From onset of a threat, quickly pivot to investigate a security signal in Datadog. Leverage the deep observability data provided by Application Security and APM distributed tracing, in one view, to resolve application issues. Analyze attack flows, view flame graphs, and review correlated trace and log data to pinpoint application vulnerabilities. Eliminate context switching by flowing through application data into remediation steps, all within the same panel.
```

> With Application Security, you can create queries to cut through the noise of continuous trace data, and begin focusing on securing and protecting your environment.

Author: ```suggestion
With Application Security, you can cut through the noise of continuous trace data to focus on securing and protecting your environment.
```

> With Application Security, you can create queries to cut through the noise of continuous trace data to focus solely on securing and protecting your environment as threats arise.

Author: ```suggestion
With Application Security, you can create queries to cut through the noise of continuous trace data, and begin focusing on securing and protecting your environment.
```

## Configure your environment

Powered by provided [out-of-the-box rules][3], Application Security detects threats without manual configuration. If you're already using Datadog [APM][1], setup only requires setting one environment variable to get started.

> To start configuring your environment to detect threats with Application Security, follow the [Getting Started documentation][5]. Once configured, you can begin investigating and remediating security signals in the [Security Signals Explorer][7].

Reviewer 2: ```suggestion
To start configuring your environment to detect threats with Application Security, follow the [Getting Started documentation][5]. Once Application Security is configured, you can begin investigating and remediating security signals in the [Security Signals Explorer][7].
```
